x-pack/test_serverless/functional/test_suites/security/common_configs/config.group6.ts

```diff
@@ -15,7 +15,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     testFiles: [
       require.resolve('../../common/discover/embeddable'),
       require.resolve('../../common/discover/x_pack'),
-      require.resolve('../../common/discover_ml_uptime/discover'),
+      // this test suite is failing for Security project after switch to SAML admin role
```

Comment:
Probably the issue is related to 'admin' role access. Skipping the test since it is working for other 2 projects and test file already has `if` logic for Security. I suggest Security project to have its own test for this functionality with appropriate role to be used.

Reply:
Can we create an issue for the failing test with some details like the failure message? And then link to that issue from the comment here? Just want to make sure that this does not get lost.

Reply:
Sure, I will add a link